spec/unit/validator_spec.rb

```diff
@@ -395,6 +395,12 @@ def podspec_path(name = 'JSONKit', version = '1.4')
           (!!target_definition.uses_frameworks?).should == false
           # rubocop:enable Style/DoubleNegation
         end
+
+        it 'inhibits warnings for all pods except the one being validated' do
+          podfile = @validator.send(:podfile_from_spec, :ios, '5.0')
+          target_definition = podfile.target_definitions['App']
+          target_definition.should.not.inhibits_warnings_for_pod?('JSONKit')
```

Comment:
should also test that other pods warnings are inhibited

Reply:
done